packages/zmarkdown/common.js

```diff
@@ -127,34 +120,41 @@ const zmdParser = (config, target) => {
         vfile.fail(`Markdown AST too complex: tree depth > ${config.maxNesting}`)
       }
     })
-}
 
-function getLatexProcessor (remarkConfig, rebberConfig, target) {
-  remarkConfig.noTypography = true
+  if (extraRemarkPlugins && extraRemarkPlugins.length > 0) {
+    for (const record of extraRemarkPlugins) {
+      if (!record.check || record.check(config)) {
+        if (record.option) {
```

Comment:
In the README you wrote that passing `null` as option means no configuration for the plugin. Is this really what we want? What if a plugin has a default config and passing `null` disables it? In this case we couldn't pass `null`.

I think there is some confusion between the semantics of `null` and `undefined`. A missing config would be `undefined`, it would use a default value if any. Explicitly passing `null` should still be possible.

Reply:
I still think this condition is incorrect. I think you are confusing `null` with `undefined`.

Imagine the following plugin:

```
(config) => {
  if (typeof config === 'undefined') config = defaultConfig // this is basically (config = defaultConfig) => {}
  doSomething(config)
}
```

How do I pass `null` to `doSomething`? (`if (record.option)` will be `false` when `record.option === null`, which is why this condition is not correct.)

Reply:
In other words: the config should un*defined* unless you *define* some config.

Defining the config as `null` means that `null` is the config, it is defined as `null` and therefore it is not undefined.

Reply:
There are plenty of other cases.

`if (config) useConfig(config)` doesn't let you use `null` or `false` or `0` or `''` as config. All of these values could be used by plugins authors.

Reply:
Indeed, I have do the change.